code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm

```diff
@@ -61,7 +61,7 @@
 	..()
 	// Set up wordfilter
 	speech_filter = new
-	speech_filter.addPickReplacement("\\b(asshole|comdom|shitter|shitler|retard|dipshit|dipshit|greyshirt|nigger|security|shitcurity)",
+	speech_filter.addPickReplacement("\\b(asshole|condom|shitter|shitler|retard|dipshit|dipshit|greyshirt|nigger|security|shitcurity)",
```

Comment:
🤔

Reply:
fixing others past typos. wtf is a comdom

Reply:
it's not a typo, comdom is the original term and likely a bastardized portmanteau of "command" and "dominant"